app/DynamicsAdapter/Fams3Adapter.Dynamics.Test/SearchRequest/SearchRequestServiceTest.cs

```diff
@@ -435,5 +460,47 @@ public async Task with_correct_bankInfomationId_upload_worksafebcClaim_should_su
 
             Assert.AreEqual("compensationClaimNumber", result.ClaimNumber);
         }
+
+        [Test]
+        public async Task upload_ICBCClaimEntity_should_success()
+        {
+            var claim = new ICBCClaimEntity()
+            {
+                ClaimNumber = "icbcClaim",
+                SearchRequest = new SSG_SearchRequest() { SearchRequestId = testId },
+                Person = new SSG_Person() { PersonId = testPersonId }
+
+            };
+
+            var result = await _sut.CreateInsuranceClaim(claim, CancellationToken.None);
+
+            Assert.AreEqual("icbcClaim", result.ClaimNumber);
+        }
+
+        [Test]
+        public async Task upload_SimplePhoneNumber_should_success()
+        {
+            var phone = new SSG_SimplePhoneNumber()
+            {
+                PhoneNumber="phone"
+            };
+
+            var result = await _sut.CreateSimplePhoneNumber(phone, CancellationToken.None);
+
+            Assert.AreEqual("phone", result.PhoneNumber);
+        }
+
+        [Test]
+        public async Task upload_InvolvedParty_should_success()
+        {
+            var party = new SSG_InvolvedParty()
+            {
+                OrganizationName = "party"
+            };
+
+            var result = await _sut.CreateInvolvedParty(party, CancellationToken.None);
+
+            Assert.AreEqual("party", result.OrganizationName);
+        }
     }
```

Comment:
We need to write more tests to demonstrate failure scenarios or missing properties.

Reply:
This file is already 100% covered.